functions/src/constellation.ts

```diff
@@ -9,85 +8,72 @@ let currentInstance: Constellation;
  * Handles all communication with Constellation throughout the project.
  */
 export class Constellation {
-    libURL: string
-
-    /**
+  /**
    * @constructor
    */
-    constructor() {
-      dag4.network.config({
-        id: config().dag_network.id,
-        beUrl: config().dag_network.be_url,
-        lbUrl: config().dag_network.lb_url,
-      });
-
-      dag4.di.useFetchHttpClient(fetch);
-
-      this.libURL = dag4.network.getNetwork().lbUrl;
-    }
+  constructor() {
+    dag4.di.useFetchHttpClient(fetch);
```

Comment:
You still need to configure the network in this version of dag4. 

```
dag4.account.connect({ networkVersion: '1.0', testnet: false });
```

Reply:
Ok, adding it. I tested with a test bot though and it worked fine

Reply:
network v1 is default so that's why it works. Adding the config explicitly here will let us switch to network 2.0 easily.